doc/doc_02_cap_01.rst

```diff
@@ -133,7 +133,7 @@ Nello specifico dei servizi digitali, che appunto vengono erogati su Internet, i
 
-Per realizzare un servizio digitale, come detto, è necessario progettare e realizzare i tre strati; lo strato di logica applicativa offre la sua API affinchè chi sviluppa lo strato di presentazione all'utente possa utilizzarla come se la logica applicativa fosse una libreria; estendendo, se vari sistemi esportano le proprie logiche applicative come API, la logica di presentazione può utilizzarle insieme, mischiandole (*mash-up*), esattamente come nello sviluppo di software moderno si programma riutilizzando le librerie offerte nel linguaggio di programmazione, sistema operativo, ecc. Quando il servizio digitale è erogato su Internet, e prevalentemente sul Web che si basa sul protocollo HTTP,) si parla di Web API. Per le Web API l'erogatore potrebbe decidere di rendere disponibile l'API non soltanto a chi sviluppa la logica di presentazione, ma "aperta" anche ad altre organizzazioni che volessero collaborare con l'erogatore, in questo caso si parla di Open API . In molti contesti, con abuso di nomenclatura, ma intuitivamente chiaro, i due termini vengono confusi e considerati sinonimi (dato che l'apertura è spesso associata al Web/Internet).
+Per realizzare un servizio digitale, come detto, è necessario progettare e realizzare i tre strati; lo strato di logica applicativa offre la sua API affinché chi sviluppa lo strato di presentazione all'utente possa utilizzarla come se la logica applicativa fosse una libreria; estendendo, se vari sistemi esportano le proprie logiche applicative come API, la logica di presentazione può utilizzarle insieme, mischiandole (*mash-up*), esattamente come nello sviluppo di software moderno si programma riutilizzando le librerie offerte nel linguaggio di programmazione, sistema operativo, ecc. Quando il servizio digitale è erogato su Internet (e prevalentemente sul Web che si basa sul protocollo HTTP) si parla di Web API. Per le Web API l'erogatore potrebbe decidere di rendere disponibile l'API non soltanto a chi sviluppa la logica di presentazione, ma "aperta" anche ad altre organizzazioni che volessero collaborare con l'erogatore, in questo caso si parla di Open API. In molti contesti, con abuso di nomenclatura, ma intuitivamente chiaro, i due termini vengono confusi e considerati sinonimi (dato che l'apertura è spesso associata al Web/Internet).

```

Comment:
Da "affinchè" a "affinché"; da "erogato su Internet, e prevalentemente sul Web che si basa sul protocollo HTTP,) si parla di Web API" a "erogato su Internet (e prevalentemente sul Web che si basa sul protocollo HTTP) si parla di Web API"; da "Open API . In" a "Open API. In"

Reply:
ok